packages/test-utils/src/wrapper.js

```diff
@@ -428,7 +428,7 @@ export default class Wrapper implements BaseWrapper {
   overview(): void {
```

Comment:
The `overview()` function uses `isVisible()` in L439.

Reply:
Nevermind... `overview` is deprecated as well. I was just confused because I recently saw the function implemented for the first time

Reply:
What do you think about this deprecation?

If people find it very useful, I guess we can keep it - the idea was to thin down the API and really focus on providing a clean experience there.

Reply:
I am currently working on my first real project with Vue. Without a lot of experience, you stumble across a number of hurdles when testing. For this, the method was helpful a few times, even if it was never part of my final tests. I used it for debugging purposes.

Up to this point I would surely have done it without this method. I think to make a decision you would have to ask various other people.

Limiting the API to the bare minimum is definitely a good decision.